baby-liminal-extension/src/lib.rs

```diff
@@ -5,13 +5,13 @@ compile_error!(
     "Features `ink` and `runtime` are mutually exclusive and cannot be enabled together"
 );
 
-#[cfg(feature = "ink")]
-pub mod api;
 pub mod error_codes;
 pub mod extension_ids;
+#[cfg(feature = "ink")]
+pub mod frontend;
```

Comment:
Too much time spent with SNARKs I think :) I liked `api` more but I'll accept this as well.

Reply:
yeah, but there is also the second part, which in this lore will be just `backend` (sounds better than `impl` :P)